### PR TITLE
Scy/Display for Quiz Attempts In Progress /US136841/US136842

### PR DESCRIPTION
**Pull request by Scy-D2L**
- Attempts in progress hide the "feedback" and "score" fields on questions
- Questions that have not been saved now show "not saved" instead of nothing
- Adds `Not Saved` langterm
- Fixes Vdiff for `attempt-result`
- Fixes warnings for `footer-events` tests

image

Handles these stories:
[US136841](https://rally1.rallydev.com/#/?detail=/userstory/628621870201&fdp=true): [CE][Quizzing][UI] Update question "Save Time" for quiz attempts in progress
[US136842](https://rally1.rallydev.com/#/?detail=/userstory/628622207023&fdp=true): [CE][Quizzing][UI] Hide question score/feedback for quiz attempts in progress

Best viewed with 'hide whitespace'.

### CODE REVIEW

File: test/consistent-evaluation-footer-events.test.js [hunk 1]
```diff
@@ -120,18 +120,18 @@ describe('d2l-consistent-evaluation-footer event tests', () => {
 	it('should not show publish or save button', function() {
 		fixture(noPermissionNotPublishedComponent).then(el => {
 			const publishButtion = getPublishButton(el);
-			expect(publishButtion).to.be.null();
```
Comment (Scy-D2L) [inline test/consistent-evaluation-footer-events.test.js:123]: These kept throwing warnings on every test run, verified that this [isn't supposed to be a function.](https://www.chaijs.com/api/bdd/#null)

File: components/left-panel/quizzing/consistent-evaluation-quizzing-attempt.js [hunk 2]
```diff
@@ -233,7 +233,10 @@ export class ConsistentEvaluationQuizzingAttempt extends SkeletonMixin(RtlMixin(
 						question-number="${ifDefined(question.questionNumber)}"
 						question-href="${ifDefined(question.questionHref)}"
 						question-response-href="${ifDefined(question.questionResponseHref)}"
+						saved-time="${ifDefined(question.savedTime)}"
+						?hide-feedback=${!this._quizAttemptInfo.complete}
```
Comment (jeroach) [inline components/left-panel/quizzing/consistent-evaluation-quizzing-attempt.js:237]: Are `hide-feedback` and `hide-score` always going to be dependent on the same thing? 

`!this._quizAttemptInfo.complete`

If so, why not just have a single variable? e.g. `hide-feedback-and-score`

Comment (Scy-D2L) [reply components/left-panel/quizzing/consistent-evaluation-quizzing-attempt.js]: Good point 👍🏼

**Review by jeroach — APPROVED** [review]
lgtm!

**Pull request merged**